emails/tests/views_tests.py

```diff
@@ -135,6 +145,67 @@ def test_sns_message_with_spam_bounce_sets_auto_block_spam(self):
         assert profile.auto_block_spam == True
 
 
+class SnsMessageTest(TestCase):
+    def setUp(self) -> None:
+        self.bucket = 'test-bucket'
+        self.key = '/emails/objectkey123'
+
+        patcher = patch('emails.views._get_address', side_effect=ObjectDoesNotExist())
+        patcher.start()
+        self.addCleanup(patcher.stop)
+
+    @patch('emails.views.remove_message_from_s3')
```

Comment:
All these test make sure we call `utils.remove_message_from_s3` but we're not testing that function. We could add tests around that function too, or make these integration tests using `moto` to make sure the expected aws api functions are called too.

Reply:
We can do this in a follow-up.